### DISCUSSION TITLE
What is Coff2IEEE?

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#1000] [General]
Due to #994, I can't see the whole article on https://orangec.readthedocs.io/en/latest/general/Coff2IEEE/.

The output of `Coff2IEEE --help` doesn't provide much information, too.

I guess it's a tool to convert MSVC object file and MSVC import library/static library (that has the extension .lib) to Orange C format.

Please correct me if I'm wrong and please give me more information about it. Thank you.

**Comment by LADSoft** [comment]
yeah basically I developed this to support someone who was trying to use an MSVC library; it converts libraries/object files from msvc format to orange co format.   I'm not sure how well it works with the current MSVC object files though as I haven't much maintained it
